data/fixtures/scopes/textual/boundedParagraph4.scope

```diff
@@ -0,0 +1,43 @@
+aaa(bbb) {
+  ccc
+}
+---
+
+[#1 Content] =
+[#1 Removal] =
+[#1 Domain] = 0:0-2:1
+  >----------
+0| aaa(bbb) {
+1|   ccc
+2| }
+   -<
+
+[#1 Insertion delimiter] = "\n\n"
+
+
+[#2 Content] =
+[#2 Domain] = 0:4-0:7
+      >---<
+0| aaa(bbb) {
+
+[#2 Removal] = 0:0-1:0
+  >----------
+0| aaa(bbb) {
+1|   ccc
+   <
```

Comment:
Hmm. I'm not sure we want to surface this one at all. If we do, the content and removal range have a very strange relationship 😅